Code/src/megadesk.cpp

```diff
@@ -595,13 +633,19 @@ void parseData(byte command, uint16_t position, uint8_t push_addr)
 
 void loop()
 {
+  readButtons();
```

Comment:
Previously if the Lin startup failed, we'd continue to read nothing from the motors in linBurst(). Button-presses were ignored, no feedback was heard.
Moving readButtons() ahead of linBurst() allows feedback, and the possibility of reset like this when motors aren't responding.